### DISCUSSION TITLE
할 일 생성 모달에 컬럼 변경 리스트 추가

### DISCUSSION THREAD

**Discussion by wjsdncl** [Part3-Team15/taskify#183] [Ideas]
할 일 수정에 있는 컬럼 변경하는 상태를 할 일 생성에도 추가하는거는 어떨까요?
![image](https://github.com/Part3-Team15/taskify/assets/89517903/68c007a2-79c5-48b9-a31c-5b9d6192fac6)

**Comment by un0211** [comment]
저도 이 생각 했어요!!
오히려 그대로 재사용하면 되고, 기본값으로 원래 만들려했던 컬럼 넣어주면 유저가 사용할 때도 오히려 편할 것 같아요

**Comment by oceanlee-seoul** [comment]
그러게요??? 그렇게 하는 게 더 사용성도 좋겠네요!!!
